packages/venia-concept/webpack.config.js

```diff
@@ -45,6 +45,17 @@ module.exports = async env => {
     const storeConfigData = await getStoreConfigData();
     const { availableStores } = await getAvailableStoresConfigData();
 
+    /**
+     * Loop the available stores when there is provided STORE_VIEW_CODE
+     * in the .env file, because should set the store name from the
+     * given store code instead of the default one.
+     */
+    const availableStore = process.env.STORE_VIEW_CODE
+        ? availableStores.find(
+              store => store.code === process.env.STORE_VIEW_CODE
+          )
+        : undefined;
```

Comment:
We can also simplify this, since `Array.find` will return `undefined` if it does not find an entry that matches the predicate.

```suggestion
    const availableStore = availableStore.find({ code } => code === process.env.STORE_VIEW_CODE);
```